content/source/docs/cloud/api/team-access.html.md

```diff
@@ -28,6 +28,8 @@ The team access APIs are used to associate a team to permissions on a workspace.
 
 -> **Note**: A `team-workspace` resource represents a team's _local_ permissions on a specific workspace. Teams can also have _organization-level_ permissions that grant access to workspaces, and Terraform Cloud uses whichever access level is higher. (For example: a team with the "manage workspaces" permission has admin access on all workspaces, even if their `team-workspace` on a particular workspace only grants read access.) For more information, see [Managing Workspace Access](../users-teams-organizations/teams.html#managing-workspace-access).
 
+Any member of an organization can view team access relative to their own team memberships, this includes secret teams of which they are a member. Organization owners and workspace admins can modify team access or view the full set of secret team accesses. The organization token and the owners team token can act as an owner on these endpoints. ([More about permissions.](/docs/cloud/users-teams-organizations/permissions.html))
```

Comment:
```suggestion
Any member of an organization can view team access relative to their own team memberships -- this includes secret teams of which they are a member. Organization owners and workspace admins can modify team access or view the full set of secret team accesses. The organization token and the owners team token can act as an owner on these endpoints. ([More about permissions.](/docs/cloud/users-teams-organizations/permissions.html))
```

Reply:
Alternative option of a `hyphen` if you had wanted this point to stand out a bit more but still remain on a continuum from prior part of the sentence.